felix/environment/feature_detect_test.go

```diff
@@ -289,7 +289,7 @@ func TestIptablesBackendDetection(t *testing.T) {
 				Ip6Nft:    -1,
 				Ip4Nft:    -1,
 			},
-			"legacy",
+			"nft",
```

Comment:
I don't think the default should be changing in these tests.  If `iptables-nft` fails to run then I think we should stick with `legacy` mode